### ISSUE TITLE
updating peers should force resync

### ISSUE THREAD

**Issue by nionis** [amark/gun#499]
At the moment, updating peers will not immediately resync due to lazily evaluated sockets, perhaps keep lazy but force resync when peers are updated

**Comment by amark** [comment]
Yes... still in the "need to do" list. Thanks for adding this, @nionis , I hope you are doing well!
